src/node_url.cc

```diff
@@ -1681,8 +1680,6 @@ void Parse(Environment* env,
   } else if (error_cb->IsFunction()) {
     Local<Value> argv[2] = { undef, undef };
     argv[ERR_ARG_FLAGS] = Integer::NewFromUnsigned(isolate, url.flags);
-    argv[ERR_ARG_INPUT] =
-      String::NewFromUtf8(env->isolate(), input).ToLocalChecked();
     error_cb.As<Function>()->Call(context, recv, arraysize(argv), argv)
         .FromMaybe(Local<Value>());
```

Comment:
Having `argv` becomes unnecessary here – you can simplify this to 

```c++
  Local<Value> flags = Integer::NewFromUnsigned(isolate, url.flags);
  USE(error_cb.As<Function>()->Call(context, recv, 1, &flags));
```

(`USE()` is generally preferable to doing a dummy `.FromMaybe()`).

Reply:
Done. Thanks again!